core/Core.rdf

```diff
@@ -236,7 +261,7 @@
 		<iof-av:adaptedFrom>APICS 14 ed., 2013, term by the same name; DoD Standard Practice, Identification Marking of US Military Property (MIL-STD-130N Nov. 2012) https://dodprocurementtoolbox.com/cms/sites/default/files/resources/2016-03/MIL-Std130N_Ch1_4.pdf, term by the same name</iof-av:adaptedFrom>
 		<iof-av:counterExample>a portion of material; a piece of glass; a rod of aluminum; a roll of aluminum; an engine block</iof-av:counterExample>
 		<iof-av:explanatoryNote>Although the term is polysemous and used in a number of other domains beyond manufacturing, it is introduced here as a covering term for any man-made artifact that satisfies the conditions provided, and independent of modality. We expect various subclasses of assembly to be introduced in future along with more precise heuristics for the various modalities in which they exist.</iof-av:explanatoryNote>
-		<iof-av:firstOrderLogicAxiom>LA1: Assembly(x) → MaterialArtifact(x) ∧ ∃c,∃c&apos;(MaterialComponent(c) ∧ MaterialComponent(c&apos;) componentPartOfAtAllTimes(c,x) ∧ componentPartOfAtAllTimes(c&apos;,x) ∧ ¬(c=c&apos;∨ (componentPartOfAtAllTimes(c,c&apos;) ∨ componentPartOfAtAllTimes(c&apos;,c))))</iof-av:firstOrderLogicAxiom>
+		<iof-av:firstOrderLogicAxiom>LA1: Assembly(x) → MaterialArtifact(x) ∧ ∃c,∃c1(MaterialComponent(c) ∧ MaterialComponent(c1) componentPartOfAtAllTimes(c,x) ∧ componentPartOfAtAllTimes(c1,x) ∧ ¬(c=c1 ∨ (componentPartOfAtAllTimes(c,c1) ∨ componentPartOfAtAllTimes(c1,c))))</iof-av:firstOrderLogicAxiom>
```

Comment:
Subformula "MaterialComponent(c1) componentPartOfAtAllTimes(c,x)" is missing a connective.